### PR TITLE
Fix image processing: use pixel ratio in subprocesses

### PR DESCRIPTION
**Pull request by j0tunn**

### CODE REVIEW

**Comment by j0tunn** [comment]
@sipayRT @tormozz48

**Comment by tormozz48** [comment]
🆗

File: test/unit/browser/existing-browser.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const ExistingBrowser = require('../../../lib/browser/existing-browser');
+
+describe('browser/existing-browser', () => {
+    it('should use pixel ratio if it set in calibration', () => {
+        const calibration = {usePixelRatio: true};
+        const browser = new ExistingBrowser('sessionId', {some: 'config'}, calibration);
+
+        assert.equal(browser.usePixelRatio, true);
```
Comment (sipayRT) [inline test/unit/browser/existing-browser.js]: `assert.isTrue(browser.usePixelRatio);`?

**Comment by sipayRT** [comment]
ok

**Pull request merged**